internal/exec/exec_public_test.go

```diff
@@ -82,14 +82,14 @@ func (suite *ExecManagerPublicTestSuite) TestRunCmdInDirWithDebug() {
 
 	em := suite.NewTestExecManager()
 
-	err := em.RunCmdInDir("echo", []string{"-n", "foo"}, "/tmp")
+	_, err := em.RunCmdInDir("echo", []string{"-n", "foo"}, "/tmp")
 	assert.NoError(suite.T(), err)
 }
 
 func (suite *ExecManagerPublicTestSuite) TestRunCmdInDirReturnsError() {
 	em := suite.NewTestExecManager()
 
-	err := em.RunCmdInDir("invalid", []string{"foo"}, "/tmp")
+	_, err := em.RunCmdInDir("invalid", []string{"foo"}, "/tmp")
```

Comment:
whats the. main reason returning string, error for these implementations?   Does't look like you're using it for the most part.

Reply:
Yeah, that's fine. All good with me.